package.json

```
@@ -3,18 +3,18 @@
   "version": "1.0.0",
   "description": "",
   "scripts": {
-    "test": "react-scripts test",   
+    "test": "react-scripts test",
     "start": "parcel src/index.html"
   },
   "author": "",
   "license": "ISC",
-  "dependencies": {      
+  "dependencies": {
     "bootstrap": "^5.3.2",
+    "prop-types": "^15.8.1",
     "react": "^18.2.0",
-    "react-bootstrap": "^2.9.2",
+    "react-bootstrap": "^2.10.1",
     "react-dom": "^18.2.0",
-    "react-scripts": "^5.0.1",
-    "prop-types": "^15.8.1"
+    "react-scripts": "^5.0.1"
```

Delete this too because this project used parcel to build not react scripts
```suggestion
    "react-scripts": "^5.0.1"
```
You also include this in the `devDependencies`
```
 "parcel": "^2.11.0",
 ```
 That way parcel is installed as part of the node modules.